Add `triangles` news post (#7374)
ppy/osu-wiki: news/2022-05-24-triangles.md

- **Community vote runner-up:** 3 months of osu!supporter
- **Community vote winner:** 6 months of osu!supporter + *Community Pick* profile badge

![Community pick profile badge](/wiki/shared/news/2022-05-24-triangles/community@2x.png)

Players have a chance to earn something too! Once the playlists are closed, whoever has the highest score on each leaderboard will receive a minor prize too:

- **Easy/Normal/Hard/Insane playlist leaders:** 1 month of osu!supporter each
- **Expert playlist leader:** 1 month of osu!supporter + *triangle aficionado* profile badge

![triangle aficionado profile badge](/wiki/shared/news/2022-05-24-triangles/aficionado@2x.png)

---

Once *everything above* is finalized, we'll announce the results in a **livestream showcase of the finalists** with commentary from some of the judges!

In case one team wins multiple categories, we'll combine their profile badges accordingly. Sorry profile badge collectors, you only receive one.

## Rules

Surprising absolutely nobody, this contest has RULES:

> - **Submission must including the following:**

Reviewer: ```suggestion
- **Submission must include the following:**
```

---
layout: post
title: "triangles"
date: 2022-05-24 23:00:00 +0000
---

Mappers and storyboarders unite in osu!'s newest contest: **triangles**!

![](https://assets.ppy.sh/contests/147/header.jpg)

[**lazer**](https://www.youtube.com/watch?v=ut_SOreClcc) is the next major update to osu!. If you [try it for yourself](https://github.com/ppy/osu#running-osu), you'll notice that osu!'s classic theme song has been replaced by *[cYsmix](https://osu.ppy.sh/beatmaps/artists/2) - triangles*:

<audio controls>
    <source src="https://assets.ppy.sh/artists/2/Songs/cYsmix%20-%20triangles.mp3" type="audio/mpeg">
</audio>

**triangles** is a mapping and storyboarding contest using lazer's theme song, and the winning submission will be **bundled with all new installations of lazer** (along with other cool prizes!).

There's a lot to explain. Let's dive in.

> ---
> 

Reviewer: there's a heading below this

```suggestion
```

- A **triangle-themed** beatmap spread...
  - of the song *[cYsmix](https://osu.ppy.sh/beatmaps/artists/2) - triangles*
  - following the [ranking criteria](/wiki/Ranking_Criteria) 
  - with **FIVE DIFFICULTIES TOTAL**: Easy, Normal, Hard, Insane, and Expert
- A **triangle-themed** storyboard

**Submit your team name and member list to [this form](https://docs.google.com/forms/d/e/1FAIpQLSeExd07kLYqp-Mx-lM7Wwxii_qidSUasAlMi3HUHUsdfVSUyg/viewform?usp=sf_link).** Once confirmed, your team will be published on the [*triangles* contest wiki page](/wiki/Contests/triangles_Contest)!

Unlike last year's [A Labour of Love](https://osu.ppy.sh/home/news/2020-11-30-a-labour-of-love) contest, entries this time around will remain **anonymous** during the judging phase. This means you should **NOT** upload your submission to BSS or share details about it until *after* judging is completed.

## Schedule

- **May 24 - July 18:** Contest is open for submissions [here](https://osu.ppy.sh/community/contests/147)!
- **July 19 - July 31:** Submissions are evaluated by a team of judges.
- **August 01 - August 15:** Top beatmaps are published for community vote and lazer playlists.

> - **August 20:** Live stream announcement of contest results!

Reviewer: ```suggestion
- **August 20:** Livestream announcement of contest results!
```

## Prizes

> Following the 3-sided theme of **triangles**, we have prizes for 3 types of people on osu!: **mappers**, **storyboarders**, and **competitive players!**

Reviewer: ```suggestion
Following the 3-sided theme of **triangles**, we have prizes for 3 types of people on osu!: **mappers**, **storyboarders**, and **competitive players**!
```

**Mappers** and **storyboarders** are in charge of judging. Each of these groups will evaluate their area of expertise AND the overall beatmap package — all while considering the contest's **triangle** theme.

Prizes will be awarded to all members of the highest scoring team in each category:

### Mapper panel

- **Runner-up:** 3 months of osu!supporter 
- **Winner:**  6 months of osu!supporter + *Mappers' Pick* profile badge

![Mappers' Pick profile badge](/wiki/shared/news/2022-05-24-triangles/mappers@2x.png)

### Storyboarder panel

- **Runner-up:** 3 months of osu!supporter 
- **Winner:**  6 months of osu!supporter + *Storyboarders' Pick* profile badge

![Storyboarders' Pick profile badge](/wiki/shared/news/2022-05-24-triangles/storyboarders@2x.png)

### Overall prizes

This category is based on how members of both Mapper and Storyboarder panels evaluate the full beatmap package.

- **Runner-up:**
  - 6 months of osu!supporter
  - *Overall runner-up* profile badge
- **Winner:**
  - 12 months of osu!supporter
  - *Overall winner* profile badge

>   - *Elite Mapper*/*Elite Storyboarder* user titles

Reviewer: these tend to be bolded in the wiki and i do think it makes it stand out more

```suggestion
  - **Elite Mapper** and **Elite Storyboarder** user titles
```

  - Submission bundled with new lazer installations (after reaching Ranked status)

![Overall runner-up profile badge](/wiki/shared/news/2022-05-24-triangles/overallrunnerup@2x.png)

![Overall winner profile badge](/wiki/shared/news/2022-05-24-triangles/overallwinner@2x.png)

### Community vote

Top beatmaps will be published for a **community vote** and featured in **5 lazer playlists** (one per difficulty level). At this time, teams may reveal their beatmaps to the public.

To participate in the community vote, you will need to play through any of the relevant playlists on lazer. This ensures anyone voting has actually considered all of the options!

- **Community vote runner-up:** 3 months of osu!supporter
- **Community vote winner:** 6 months of osu!supporter + *Community Pick* profile badge

![Community pick profile badge](/wiki/shared/news/2022-05-24-triangles/community@2x.png)

Players have a chance to earn something too! Once the playlists are closed, whoever has the highest score on each leaderboard will receive a minor prize too:

- **Easy/Normal/Hard/Insane playlist leaders:** 1 month of osu!supporter each

> - **Expert playlist leader:** 1 month of osu! supporter + *triangle aficionado* profile badge

Reviewer: ```suggestion
- **Expert playlist leader:** 1 month of osu!supporter + *triangle aficionado* profile badge
```

![triangle aficionado profile badge](/wiki/shared/news/2022-05-24-triangles/aficionado@2x.png)

---

> Once *everything above* is finalized, we'll announce the results in a **live stream showcase of the finalists** with commentary from some of the judges!

Reviewer: ```suggestion
Once *everything above* is finalized, we'll announce the results in a **livestream showcase of the finalists** with commentary from some of the judges!
```

In case one team wins multiple categories, we'll combine their profile badges accordingly. Sorry profile badge collectors, you only receive one.

## Rules

Surprising absolutely nobody, this contest has RULES:

- **Submission must including the following:**
  - A beatmap spread of *cYsmix - triangles* with **FIVE DIFFICULTIES TOTAL**: Easy, Normal, Hard, Insane, and Expert. [Download the beatmap template here!](https://assets.ppy.sh/artists/2/Songs/cYsmix%20-%20triangles.osz)

>   - A storyboard with all storyboarded content **under ~1 MB total** (including the `.osb` file and difficulty-specific storyboarding).

Reviewer: is this supposed to imply *storyboarded content only* (minus video, minus hit objects), or the whole mapset folder size? for the latter, it would be clearer to just say "the whole beatmap folder must be under ~1 MB total". the wording is a little suspiciously specific atm

Author: woops removed

- **Submission must be in `.osz` format.**
- **Beatmaps must be in *osu!* game mode only.** Comparing beatmaps of different modes in a judging scenario is unfortunately not realistic.
- **Beatmaps should abide by the [ranking criteria](/wiki/Ranking_Criteria).** While we can be forgiving for mistakes, submissions that could not be ranked without major changes will be disqualified.
- **Team name and members must be confirmed through [this form](https://docs.google.com/forms/d/e/1FAIpQLSeExd07kLYqp-Mx-lM7Wwxii_qidSUasAlMi3HUHUsdfVSUyg/viewform?usp=sf_link).**

>   - If your country does not support Google forms, [send pishifat a message](https://osu.ppy.sh/community/chat?sendto=3178418).

Reviewer: ```suggestion
  - If your country does not support Google Forms, [send pishifat a message](https://osu.ppy.sh/community/chat?sendto=3178418).
```
